website/modules/asset/ui/src/index.scss

```
@@ -26,4 +26,5 @@
 @import './scss/_not-found.scss';
 @import './scss/_table-widget';
 @import './scss/_vacancies';
+@import './scss/_search';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)